.github/workflows/pr-to-update-homebrew.yml

```diff
@@ -0,0 +1,82 @@
+name: PR to update homebrew
+
+# This action will run after the "Publish container" is succesfully completed
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run
```

Comment:
Actually just noticed that this description is no longer accurate.

Reply:
Fixed it, have to complete the ToDo list in the PR before merging.